src/tcp.rpc.client.js

```diff
@@ -0,0 +1,71 @@
+'use strict'
+
+const { promiseSleep } = require('@bitfinex/lib-js-util-promise')
+const { TaskQueue } = require('@vigan-abd/task-queue-js')
```

Comment:
please let's use a common queue system, like async.queue

Reply:
yep this one is built on top of it by myself, it's been battle proven in some project that required sequential on chain writes due to nonce

Reply:
https://github.com/vigan-abd/task-queue-js/blob/main/src/task.queue.js#L26-L35

Reply:
"@Vigan/task-queue-js": "^1.1.1",
it kinds of conflicts with standard rules:
* we should avoid ^ for non-core libraries
* low count of reviews. I understand that you made it and that's great but either we moved it under bitfinexcom or we need to use a more standard lib.

Sorry but we need to ensure consistency. Same applies to ifinex.
I try to avoid libs from my own `prdn` repo as well

Reply:
ok maybe we can fork it under bitfinexcom or I can just write same promise wrapper around async.queue here?

Reply:
let's add the promise wrapper in here for simplicity atm